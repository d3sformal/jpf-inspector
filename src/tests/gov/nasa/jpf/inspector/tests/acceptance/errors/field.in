cr bp method=mk
run && wait
print #this.#field[0]
print #this.#field[1]
print #this.field
print #this.invalid_field_name
print #this.#field[2]
print #this.#static[1]
print #this.#static[2]
del bp 1