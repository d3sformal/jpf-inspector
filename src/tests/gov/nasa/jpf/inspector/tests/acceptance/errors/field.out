INFO: JPF created and connected, SuT is started
INFO: Breakpoint hit: [1 :  state=ENABLED hits=1 hitsTotal=1 method = mk]
	In method "mk".
instanceTest (gov.nasa.jpf.inspector.tests.acceptance.sut.ExpressionTest) = null
	1 : field (int) = 5
	1 : field (int) = 5
ERR: No entry (field, local variable, parameter) with the name "invalid_field_name" exists (maybe we are inside a native method).
ERR: Invalid field index (2). "ExpressionTest" has only 2 fields.
arrayOfIntegers (int[]) = null
ERR: Static field index 2 is out of bounds. The class "ExpressionTest" has only 2 static fields.
Breakpoint with ID 1 successfully deleted.
INFO: JPF Terminating